infra/staging/main.tf

```diff
@@ -151,6 +151,66 @@ resource "google_storage_bucket_iam_member" "frontend_bucket_viewer" {
   role   = "roles/storage.objectViewer"
   member = "allUsers"
 }
+#######################################
+## Load balancer for frontend bucket ##
+#######################################
+
+// Following the guide here: https://cloud.google.com/cdn/docs/setting-up-cdn-with-bucket
+
+// Enable Compute Engine API to setup an IP address
+resource "google_project_service" "compute_engine_api" {
+  project = data.google_project.project.project_id
+  service = "compute.googleapis.com"
+}
+
+// Setup IP address for load balancer
+resource "google_compute_global_address" "call_home_frontend_ip" {
+  project = data.google_project.project.project_id
+  name = "call-home-frontend-ip"
+}
+
+// Backend bucket with CDN policy with default ttl settings
+resource "google_compute_backend_bucket" "call_home_frontend_lb" {
+  name = "call-home-frontend-lb"
```

Comment:
It would be more accurate to call this the `backend_bucket`, since that's what Terraform and the GCP console UI call it https://console.cloud.google.com/net-services/loadbalancing/advanced/backendBuckets/details/call-home-frontend-lb?project=call-home-staging

Reply:
Actually, I take this back. Naming it `backend_bucket` would be redundant with the 'kind' of thing this is, plus this is for the `frontend-lb` setup, so this is fine.